core/src/main/scala-3/cats/derived/DerivedSemigroupK.scala

```diff
@@ -0,0 +1,34 @@
+package cats.derived
+
+import cats.{Semigroup, SemigroupK}
+import shapeless3.deriving.{Const, K1}
+
+import scala.compiletime.*
+
+type DerivedSemigroupK[F[_]] = Derived[SemigroupK[F]]
```

Comment:
You're missing the implicit not found messages

Reply:
It looks like we don't have them for any of our `DerivedXXX` types? I'll create an issue for that

Reply:
Actually my editor was just being weird. I see they are there 👍

Reply:
Maybe you had an old branch